src/main/java/io/strimzi/test/container/StrimziZookeeperContainer.java

```diff
@@ -54,7 +52,7 @@ public StrimziZookeeperContainer() {
 
     @Override
     protected void doStart() {
-        this.setDockerImageName(KafkaVersionService.strimziTestContainerImageName(strimziBaseImage, strimziTestContainerImageVersion, kafkaVersion));
+        this.setDockerImageName(KafkaVersionService.strimziTestContainerImageName(kafkaVersion));
         // we need it for the startZookeeper(); and startKafka(); to run container before...
```

Comment:
I don't understand this comment. What is "it"?

Reply:
This command :)  
`sh", "-c", "while [ ! -f " + STARTER_SCRIPT + " ]; do sleep 0.1; done; " + STARTER_SCRIPT`